DTS-STN/secure-client-hub
# Update mappers to use TS

## [ADO-161733](https://dev.azure.com/VP-BD/DECD/_workitems/edit/161733)

Fixed [AB#161733](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/161733)

### Changelog
feat:update mappers to use TypeScript

### Description of proposed changes:
This PR applies the changes made in #464 given there were issues with how that PR was opened preventing tests from passing. These changes include swapping our mappers to use TypeScript and fixing up our TS implementation.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev` or `npm run build` and `npm run start`
3. Confirm the app works as expected
